### PR TITLE
Clean up Packet

### PR DESCRIPTION
**Pull request by jmarcelof**
Rename the class, some methods and attributes. Change visibility and simplify some stuff.

### CODE REVIEW

File: src/Evaluation.java [hunk 1]
```diff
@@ -31,9 +31,9 @@ public void makeHistLat() {
 		double step = (this.latencyMax() + this.latencyMin()) / (double) nDot;
 		for(int i = 0; i < nDot; i++)
 			lats[i] = (double)(i+1)*step;
-		Collections.sort(pcks, new Packet.ByLatency());
+		Collections.sort(pcks, new Package.ByLatencyComparator());
 		int i = 0;
-		for(Packet pck: pcks)
+		for(Package pck: pcks)
```
Comment (rmilfont) [inline src/Evaluation.java]: Put the brackets for all statements at this file code.

File: src/ReadFiles.java [hunk 2]
```diff
@@ -35,28 +35,28 @@ public ReadFiles(String path) {
 		Collections.sort(pcks); // ordena pelo número de identificação do pacote
 		double acceptedTraffic = 0;
 		for(int i = 0; i < pcks.size()-1; i++) {
-			Packet act = pcks.get(i);
-			Packet next = pcks.get(i+1);
+			Package act = pcks.get(i);
+			Package next = pcks.get(i+1);
 			
 			if(act.src().equals(next.src()))
```
Comment (rmilfont) [inline src/ReadFiles.java]: Put the brackets here.

File: src/Package.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        return acceptedTraffic;
+    }
+
+    void setAcceptedTraffic(double acceptedTraffic) {
+        this.acceptedTraffic = acceptedTraffic;
+    }
+
+    @Override
+    public int compareTo(Package that) {
+        if (this.id < that.id) return -1;
```
Comment (rmilfont) [inline src/Package.java]: The missing brackets here turn very easy and simple to see, but in our "best practices", what we should do in this case?

Comment (jmarcelof) [inline src/Package.java]: You're right. I'll fix only this one as I didn't really change the other two files.

Comment (jmarcelof) [inline src/Package.java]: Done

**Pull request merged**